src/common/playlists/PlayList.jsx

```diff
@@ -1,47 +1,41 @@
 import PlayThumbnail from "./PlayThumbnail";
 import { ReactComponent as ImageOops } from "images/img-oops.svg";
-import React, { Fragment, useEffect, useState } from "react";
+import React, { Fragment } from "react";
 import "react-loader-spinner/dist/loader/css/react-spinner-loader.css";
 import Loader from "common/spinner/spinner";
 import * as all_plays from "plays";
-import { useParams } from "react-router-dom";
-import useGetPlays from 'common/hooks/useGetPlays'
+import useGetPlays from "common/hooks/useGetPlays";
 
 import "./playlist.css";
 import { toSanitized } from "common/services/string";
 
 const PlayList = () => {
   const [loading, error, plays] = useGetPlays();
-  const [allPlays, setAllPlays] = useState([])
 
-  let { playid } = useParams(); // return the parameter of url
-  
   if (loading) {
-   return <Loader />;
+    return <Loader />;
   }
 
-  if (plays?.length === 0) {
+  if (plays?.length === 0 || error) {
```

Comment:
For error also the length witll zero.. isnt't it?

Reply:
Not sure about the play length but if there is an error then an error message should be shown right?